raiden/transfer/node.py

```diff
@@ -983,6 +983,8 @@ def is_transaction_effect_satisfied(chain_state, transaction, state_change):
             partner_address = state_change.partner
         elif state_change.partner == our_address:
             partner_address = state_change.participant
+        else:
+            return False
```

Comment:
This should not be added here. it's quitting the predicate early and tests added after this if will be ignored, use `partner_address = None` instead.